# Use this file to specify parameters, that do not have default values.
# YAML Files can also be helpful if you want to save parameters for different use-cases.
# In any way: Default values should go into the .if file!
#
# To generate a yaml file, use "rosrun rosinterface_handler generate_yaml /path/to/KitafController.if"
#
# Use 'key: value' pairs, e.g.
# string: 'foo'
# integer: 1234
# float: 1234.5
# boolean: true
# vector: [1.0, 2.0, 3.4]
# map: {"a": "b", "c": "d"}

verbosity: info

wheel_base: 0.51
msg_queue_size: 1
frame_id_map: world
frame_id_vehicle: vehicle_front_axle

steer_update_rate: 40
control_loop_rate: 50
velocity_loop_rate: 40
velocity_cur: 1.3
k_dis: 3.0
k_yaw: 1
k_yaw_error: 1.5
k_compensation: -0.1
error_interval: 2
k_soft: 0.0
k_shift: 0.15
slowDistance: 4
stopDistance: 2


servo_topic: /motor_interface/servo_command
motor_topic: /motor_interface/motor_command
path_topic: /path
stop_sign_topic: /stop_sign
motor_state_topic: /motor_state
currentTask_topic: /current_task

velocity_exit: 0.2
k_dis_exit: 0.6
k_yaw_exit: 1
k_yaw_error_exit: 1.5
